app/services/webhook_manager/crawl_receipt_creator.rb

```diff
@@ -0,0 +1,19 @@
+
+module WebhookManager
+   class CrawlReceiptCreator < BaseService
+    attr_reader :payment_intent, :amount, :contact_id, :payment_intent_id
+
+    def initialize(params)
+      @payment_intent = params[:payment_intent]
+      @amount = params[:amount]
+      @contact_id = payment_intent.purchaser_id
+      @payment_intent_id = payment_intent.id
+    end
+
+    def call
+      if Date.today.month == 2 && @amount >= 1000
```

Comment:
This is good enough for now, but we'd want it to be the month of February in EST. Right now it's using UTC. 

Also nits: if we could abstract these two things into constants on the top, that would be good. 

`FEBRUARY_EST = Time.now <= ... && Time.now >= ...`
`@amount >= CrawlReceipt::CRAWL_RECEIPT_MIN_AMOUNT`

I think go ahead and merge as-is though

Reply:
revising now!

Reply:
assigning the time to a constant caused the timecop freeze to fail since it doesn't re-evaluate Time.now after freezing time in the tests, using the same value in EST though instead!